02-running-grakn/01-install-and-run.md

```diff
@@ -24,6 +24,67 @@ Download the [latest release](https://grakn.ai/download?os=mac_os_x#core), unzip
 Having downloaded Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
 [tab:end]
 
+[tab:CentOS]
+
+### To download the latest release
+
+As a superuser, add the repo:
+```
+# yum-config-manager --add-repo https://repo.grakn.ai/repository/meta/grakn-core.repo
```

Comment:
@vmax do we need the `#` here?

Reply:
@sorsaffari `#` signifies that user is superuser

Reply:
I see. alright then